src/bitcoin/d/mod.rs

```diff
@@ -1394,3 +1411,85 @@ impl<'a> CachedTxGetter<'a> {
         }
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct GetMempoolEntryRes {
```

Comment:
I haven't included all the fields in the `getmempoolentry` response yet.

Reply:
No need to include the fields we won't be using (see how we only include those in every other command result).

Reply:
Do you think it's worth removing those I've already added that are not currently being used?

Reply:
Yes. Dead code is bad.